src/Core/src/Eventuous.Persistence/AggregateStore/AggregateStoreExceptions.cs

```diff
@@ -4,21 +4,21 @@
 namespace Eventuous;
 
 public class OptimisticConcurrencyException : Exception {
-    public OptimisticConcurrencyException(Type aggregateType, StreamName streamName, Exception inner)
-        : base($"Update of {aggregateType.Name} failed due to the wrong version in stream {streamName}. {inner.Message} {inner.InnerException?.Message}", inner) { }
+    public OptimisticConcurrencyException(Type aggregateType, StreamName streamName, Exception? inner)
+        : base($"Update of {aggregateType.Name} failed due to the wrong version in stream {streamName}", inner) { }
 
-    public OptimisticConcurrencyException(StreamName streamName, Exception inner)
-        : base($"Update failed due to the wrong version in stream {streamName}. {inner.Message} {inner.InnerException?.Message}", inner) { }
+    public OptimisticConcurrencyException(StreamName streamName, Exception? inner)
+        : base($"Update failed due to the wrong version in stream {streamName}", inner) { }
 }
 
-public class OptimisticConcurrencyException<T>(StreamName streamName, Exception inner) : OptimisticConcurrencyException(typeof(T), streamName, inner)
+public class OptimisticConcurrencyException<T>(StreamName streamName, Exception? inner) : OptimisticConcurrencyException(typeof(T), streamName, inner)
     where T : Aggregate;
 
 public class AggregateNotFoundException : Exception {
-    public AggregateNotFoundException(Type aggregateType, StreamName streamName, Exception inner)
-        : base($"Aggregate {aggregateType.Name} with not found in stream {streamName}. {inner.Message} {inner.InnerException?.Message}", inner) { }
+    public AggregateNotFoundException(Type aggregateType, StreamName streamName, Exception? inner)
+        : base($"Aggregate {aggregateType.Name} with not found in stream {streamName}", inner) { }
```

Comment:
this sentence does not make sense, maybe remove 'with', or make it something like 'No events for Aggregate {name} found in {streamName}' ?

Reply:
You're right - I didn't catch this while going through this. I might go through the error messages and make sure they are sensible and makes sense sometime during the weekend